packages/design-tokens/src/components/note.yml

```diff
@@ -0,0 +1,38 @@
+note:
+  padding:
+    value: "{size-px.m}"
+  border-radius:
+    value: "{border-radius.default}"
+  border-width:
+    default:
+      value: "{border-width.default}"
+    main:
+      value: "{size-px.s}"
+  border-style:
+    value: "{border-style.default}"
+  text-to-text-spacing:
+    value: "{size-px.xs}"
+  icon-to-text-spacing:
+    value: "{size-px.m}"
```

Comment:
Bei diesen eher Rem?

Reply:
ich hät gesagt nein, weil der Abstand zum Rand und zum Text so immer gleich groß bleibt

Reply:
Aber es geht doch um "text-to-text" bzw. "icon-to-text". Haben wir beim Button auch so gemacht. Der generelle Innenabstand einer Component – ja der kann in px angegeben werden.

Der vertikale Abstand sollte schon (ähnlich wie die `line-height`) proportional zum Text sein. Es soll ja ein deutlicher Abstand von Headline zu Text erkennbar sein. Der Abstand wird aber durch eine wachsende Schriftgröße "aufgefressen".

Der Abstand "icon-to-text" ist auch eher proportional. Auch hier sollte eine visuelle Abgrenzung durch Abstand erkennbar. Vgl. Button.

Wir können hier aber auch gerne nochmal im Daily um Feedback fragen, wenn wir hier uneinig sind

Reply:
![Bildschirmfoto 2024-01-11 um 13 59 55](https://github.com/mittwald/flow/assets/84317589/ebd8f6b5-b367-48a1-a16e-98ad8ca4ee4f)

achso ja ich meinte nur das der Abstand vom Icon zum Rand und zum Text so gleich ist... aber du hast recht,die lesbarkeit ist wichtiger, ich pass es an 👍